src/styles/main.scss

```diff
@@ -1,3 +1,54 @@
+@import 'header';
+@import 'card';
+@import 'cards';
+@import 'variables';
+
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: roboto, sans-serif;
+  font-size: 10px;
+  line-height: 14px;
+}
+
 body {
```

Comment:
The display: block; is not needed on the body tag. Body is a block-level element by default.